app/models/comment.rb

```diff
@@ -1,2 +1,9 @@
 class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+  before_save :comment_counter
```

Comment:
**Great job completing the major requirements. However, we have some places to add up.** 

- [ ] Kudos for adding an update method for the counter. But, it would be nice if you save it `after_save` you change rather than `before_save`. Also, do not forget to put your update method after `private`, ok.